extensions/amp-story/amp-story.md

```diff
@@ -254,53 +250,193 @@ The `bookend-config-src` value is a URL endpoint that returns data for the end s
   </amp-anim>
 </figure>
 
-#### Related articles
+#### Bookend components
+
+The bookend is made up of a variety of components. These components can be articles, call to action links, text, and more.
+
+Bookend components are specified in the `components` field of the configured JSON. See the [Example JSON response](#example-json-response) section below for an example.
+
+##### heading
+
+The `heading` component has a `text` field, which can be used to append a title to a group of articles.
+
+
+```json
+{
+  "type": "heading",
+  "text": "More to Read"
+}
+```
+
+<amp-img alt="Bookend heading component" layout="fixed"
+src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/amp-story-bookend-comopnent-heading.png" width="386" height="123">
+  <noscript>
+    <img alt="Bookend heading component" src="img/amp-story-bookend-comopnent-heading.png" />
+  </noscript>
+</amp-img>
+
+
+##### small
+
+The `small` component can be used to link to related articles. This component requires the following fields: `title`, `url`, and optionally an `image`.
+
+```json
+{
+  "type": "small",
+  "title": "This is India an the best places you should go",
+  "url": "http://example.com/article.html",
+  "image": "http://placehold.it/256x128"
+}
+```
+
+<amp-img alt="Bookend small component" layout="fixed"
+src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/amp-story-bookend-comopnent-small.png" width="379" height="192">
+  <noscript>
+    <img alt="Bookend small component" src="img/amp-story-bookend-comopnent-small.png" />
+  </noscript>
+</amp-img>
+
+##### landscape
+
+The `landscape` component can be used for alternative formats of content, like videos. This component requires the following fields: `title`, `category`, `url`, and an `image`.
+
+```json
+{
+  "type": "landscape",
+  "title": "TRAPPIST-1 Planets May Still Be Wet Enough for Life",
+  "url": "http://example.com/article.html",
+  "category": "astronomy",
+  "image": "http://placehold.it/256x128"
+}
+```
+
+<amp-img alt="Bookend landscape component" layout="fixed"
+src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/amp-story-bookend-comopnent-landscape.png" width="388" height="410">
+  <noscript>
+    <img alt="Bookend landscape component" src="img/amp-story-bookend-comopnent-landscape.png" />
+  </noscript>
+</amp-img>
+
+##### portrait
+
+The `portrait` component can be used to link to other stories. This component requires the following fields: `category`, `url`, and an `image`.
```

Comment:
Why doesn't the portrait allow a `title`?  You're accompanying image of a portrait type actually includes a title.  It seems incongruent that the other types allow title but not this one, especially if it's to link to other articles.

Reply:
The screenshot includes a `category`, but not a `title`.

@hongwei1990, can you confirm that portrait is not allowed to have a title?  It is indeed pictured this way in the mocks.

Reply:
This component was made for promoting AMP stories with a large hero story cover. To avoid the title repeat and save the component height. We hide the cover before. But since we can't predict the content type in the bookend, let's add the title section in for this card as default. The spec will follow what we had for the landscape card. If nothing gets specified in the title section of this card, we will hide it. Please reference the mock in attachment.
<img width="257" alt="screen shot 2018-05-25 at 12 28 34 pm" src="https://user-images.githubusercontent.com/33703058/40563150-993cfda6-6018-11e8-8e36-462a5686ed9f.png">

Reply:
Ok, I will add the option to add a title in another PR.